src/components/Search/Search.js

```diff
@@ -16,25 +15,28 @@ class Search  extends Component  {
 
     handleClick = (e) => {
         e.preventDefault();
-        const foundPokemon = {
-            id:Date.now(),
+        const queriedPokemon = {
+            key: Date.now(),
             ...this.state
         }
-        console.log(foundPokemon)
-        // foundPokemon will be = data.name 
+        // console.log(queriedPokemon)
+        this.props.addPokemon(queriedPokemon)
+        // if(props.addPokemon(queriedPokemon)
+        // then addPokemon(queriedPokemon)
+        // else show error
     }
 
     render() {
         return (
             <form>
-                <input 
-                    type="search" 
-                    name="foundPokemon" 
-                    className="search-bar" 
-                    placeholder="Choose your Pokemon" 
+                <input
+                    type="search"
+                    name="queriedPokemon"
+                    className="search-bar"
+                    placeholder="Choose your Pokemon"
```

Comment:
I really like this indentation, looks away better tsk for paying attention to that